### PR TITLE
WoRlD cLaSs UpDaTeS

### PR DESCRIPTION
**Pull request by mouncg**
- code formatting

- image optimization

- start of default embed color (will be continued later)

# fully wOrLd ClAsS

### CODE REVIEW

File: cogs/battling.py [hunk 1]
```diff
@@ -44,10 +44,12 @@ def selected(self):
         return self.pokemon[self.selected_idx]
 
     async def send_selection(self):
-        embed = discord.Embed()
-        embed.color = 0xF44336
+        embed = discord.Embed(color=self.bot.embed_color)
```
Comment (oliver-ni) [inline cogs/battling.py:47]: could we possibly subclass discord.Embed to set this automatically?

File: cogs/battling.py [hunk 2]
```diff
@@ -623,7 +619,10 @@ async def moveinfo(self, ctx: commands.Context, *, search: str):
             ("Priority", "priority"),
             ("Type", "type"),
         ):
-            if (v := getattr(move, x)) is not None:
+            if getattr(move, x) is not None:
+                v = getattr(
+                    move, x
+                )  # yeah, i had to remove walrus op cuz its just too bad lol
```
Comment (oliver-ni) [inline cogs/battling.py:625]: Please add the walrus back

File: cogs/pokemon.py [hunk 3]
```diff
@@ -287,7 +287,8 @@ async def create_filter(self, flags, ctx, order_by=None):
                     )
 
         if order_by is not None:
-            if (s := order_by[-1]) in "+-":
+            s = order_by[-1]
+            if order_by[-1] in "+-":
```
Comment (oliver-ni) [inline cogs/pokemon.py:291]: Please add the walrus back

File: cogs/shop.py [hunk 4]
```diff
@@ -563,7 +562,8 @@ async def buy(self, ctx: commands.Context, *args: str):
 
             pokemon.level += qty
             guild = await self.db.fetch_guild(ctx.guild)
-            if (evo := pokemon.get_next_evolution(guild.is_day)) is not None:
+            if pokemon.get_next_evolution(guild.is_day) is not None:
+                evo = pokemon.get_next_evolution(guild.is_day)
```
Comment (oliver-ni) [inline cogs/shop.py:566]: Please add the walrus back

File: cogs/shop.py [hunk 5]
```diff
@@ -57,8 +57,8 @@ async def vote(self, ctx: commands.Context):
                 value=f"Current Streak: {member.vote_streak} votes!",
                 inline=False,
             )
-
-        if (later := member.last_voted + timedelta(hours=12)) < datetime.utcnow():
+        later = member.last_voted + timedelta(hours=12)
+        if later < datetime.utcnow():
```
Comment (oliver-ni) [inline cogs/shop.py:61]: Please add the walrus back

File: cogs/trading.py [hunk 6]
```diff
@@ -136,8 +136,9 @@ def txt(x):
                             dec += 1
 
                         if (
-                            evo := pokemon.species.trade_evolution
+                            pokemon.species.trade_evolution
                         ) and pokemon.held_item != 13001:
+                            evo = pokemon.species.trade_evolution
```
Comment (oliver-ni) [inline cogs/trading.py:141]: Please add the walrus back

File: helpers/models.py [hunk 7]
```diff
@@ -269,7 +269,8 @@ def target(self):
 
     @cached_property
     def text(self):
-        if (pevo := getattr(self.target, f"evolution_{self.dir}")) is not None:
+        if getattr(self.target, f"evolution_{self.dir}") is not None:
+            pevo = getattr(self.target, f"evolution_{self.dir}")
```
Comment (oliver-ni) [inline helpers/models.py:273]: Please add the walrus back

File: cogs/battling.py (hunk 1 above)
Comment (mouncg) [reply cogs/battling.py:47]: > yeah, that's what I'm doing rn lol

File: cogs/battling.py (hunk 2 above)
Comment (mouncg) [reply cogs/battling.py:625]: Yes, I know, I will do that after

File: cogs/pokemon.py (hunk 3 above)
Comment (mouncg) [reply cogs/pokemon.py:291]: I do understand

File: cogs/shop.py (hunk 4 above)
Comment (mouncg) [reply cogs/shop.py:566]: 😂

File: cogs/shop.py (hunk 5 above)
Comment (mouncg) [reply cogs/shop.py:61]: lol

File: cogs/shop.py (hunk 4 above)
Comment (oliver-ni) [reply cogs/shop.py:566]: LOL

File: cogs/trading.py (hunk 6 above)
Comment (mouncg) [reply cogs/trading.py:141]: yessir

File: helpers/models.py (hunk 7 above)
Comment (mouncg) [reply helpers/models.py:273]: lol will do

**Pull request merged**